### DISCUSSION TITLE
Numeric Field

### DISCUSSION THREAD

**Discussion by AnthonyGerges** [OrchardCMS/OrchardCore#15223] [General]
okay so im gonna share my json code that let me create a content item of type i also created , that has fields as numeric and text , so when i create the content item , it shows me the text field , but the numeric field still empty , so any one nknow why? 
{
  "name": "Blog",
  "displayName": "Blog",
  "description": "Provides a functional Blog with CMS features.",
  "author": "The Orchard Core Team",
  "website": "https://orchardcore.net",
  "version": "1.0.0",
  "issetuprecipe": false,
  "categories": [ "default" ],
  "tags": [ "blog" ],

  // The variables are evaluated the first time they are accessed, and reused across steps
  "variables": {
    "adminUrlPrefix": "[js: configuration('OrchardCore_Admin:AdminUrlPrefix', 'Admin')]",
    "blogContentItemId": "[js:uuid()]",
    "menuContentItemId": "[js:uuid()]",
    "tagsContentItemId": "[js:uuid()]",
    "tagContentItemId": "[js:uuid()]",
    "earthTagContentItemId": "[js:uuid()]",
    "explorationTagContentItemId": "[js:uuid()]",
    "spaceTagContentItemId": "[js:uuid()]",
    "categoriesContentItemId": "[js:uuid()]",
    "categoryTravelContentItemId": "[js:uuid()]"
    // "now": "[js: new Date().toISOString()]"
  },

  "steps": [
    {
      "name": "feature",
      "enable": [
        // SaaS
        "OrchardCore.HomeRoute",
        "OrchardCore.Admin",
        "OrchardCore.Diagnostics",
        "OrchardCore.DynamicCache",
        "OrchardCore.Features",
        "OrchardCore.Navigation",
        "OrchardCore.Recipes",
        "OrchardCore.Resources",
        "OrchardCore.Roles",
        "OrchardCore.Security",
        "OrchardCore.Settings",
        "OrchardCore.Themes",
        "OrchardCore.Users",

        // Content Management
        "OrchardCore.Alias",
        "OrchardCore.AdminMenu",
        "OrchardCore.Autoroute",
        "OrchardCore.Html",
        "OrchardCore.ContentFields",
        "OrchardCore.ContentPreview",
        "OrchardCore.Contents",
        "OrchardCore.ContentTypes",
        "OrchardCore.CustomSettings",
        "OrchardCore.Deployment",
        "OrchardCore.Deployment.Remote",
        "OrchardCore.Feeds",
        "OrchardCore.Flows",
        "OrchardCore.Indexing",
        "OrchardCore.Layers",
        "OrchardCore.Lists",
        "OrchardCore.Markdown",
        "OrchardCore.Media",
        "OrchardCore.Menu",
        "OrchardCore.Placements",
        "OrchardCore.Queries",
        "OrchardCore.Queries.Sql",
        "OrchardCore.Shortcodes.Templates",
        "OrchardCore.Rules",
        "OrchardCore.Taxonomies",
        "OrchardCore.Title",
        "OrchardCore.Templates",
        "OrchardCore.Widgets",

        // Themes
        "TheBlogTheme",
        "TheAdmin",
        "SafeMode"
      ]
    },

    {
      "name": "content",
      "Data": [

        {
          "ContentItemId": "[js: uuid()]",
          "ContentType": "MarketTradingHour",
          "DisplayText": "Item1",
          "Latest": true,
          "Published": true,
          "Owner": "[js: parameters('AdminUserId')]",
          "Author": "[js: parameters('AdminUsername')]",
          "AutoroutePart": {
            "Path": "item",
            "SetHomepage": false
          },
          "HtmlBodyPart": {
            "Html": "<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Saepe nostrum ullam eveniet pariatur voluptates odit, fuga atque ea nobis sit soluta odio, adipisci quas excepturi maxime quae totam ducimus consectetur?</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius praesentium recusandae illo eaque architecto error, repellendus iusto reprehenderit, doloribus, minus sunt. Numquam at quae voluptatum in officia voluptas voluptatibus, minus!</p><p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nostrum molestiae debitis nobis, quod sapiente qui voluptatum, placeat magni repudiandae accusantium fugit quas labore non rerum possimus, corrupti enim modi! Et.</p>"
          },
          "TitlePart": {
            "Title": "Trading Hour"
          },
          "MarketTradingHour": {
            "MarketTradingHourFrom": {
              "Numeric": 10
            },

            "MarketTradingHourTo": {
              "Numeric": 12
            },
            "MarketTradingHourisDefault": {
              "Boolean": true
            },
            "MarketTradingHourName": {
              "Text": "Hello"
            },

            "MarketTradingHourColor": {
              "Text": "Red"
            }

          }
        }

      ]
    }

  

  ]
}

**Comment by daveblack101** [comment]
That should be "Value" not "Numeric":
```
"MarketTradingHourFrom": {
     "Value": 10
},
"MarketTradingHourTo": {
     "Value": 12
}
```
